index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PIX TECH</title>
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <header>
+      <h1 class="logo">
+        <a href="http://pixtech.pixel-hearts.co.jp/#first-view">
+          <img src="img/logo_sticky.png" alt="PIX TECH" />
+        </a>
+      </h1>
+      <nav>
+        <ul class="navList">
+          <li class="navListItem"><a href="#">PIX TECHの特徴</a></li>
+          <li class="navListItem"><a href="#">PIX TECHの思い</a></li>
+          <li class="navListItem"><a href="#">カリキュラム</a></li>
+          <li class="navListItem"><a href="#">コースの紹介</a></li>
+          <li class="navListItem"><a href="#">講師の紹介</a></li>
+          <li class="navListItem"><a href="#">よくある質問</a></li>
+          <li class="navListItem"><a href="#">説明会へ申し込む</a></li>
```

Comment:
コンポーネント思考でクラスを当ててほしいです。
```suggestion
          <li class="navListItem"><a href="#" class="button">説明会へ申し込む</a></li>
```

Reply:
クラスはタグの後ろであることが好ましいと思っていたのですが、このような記述の仕方があるとは知りませんでした。ご教示あるがとうございます。また、btnではなく、buttonに修正します。